packages/calypso-e2e/src/lib/blocks/videopress-block.ts

```diff
@@ -0,0 +1,70 @@
+import { Page } from 'playwright';
+import { EditorComponent } from '../components';
+
+/**
+ * Represents the VideoPress block.
+ */
+export class VideoPressBlock {
```

Comment:
The VideoPress spec has been implemented as as regular block instead of a block smoke testing flow. This is because when I initially tried to implement with the block smoke testing pattern, but ran into difficulty because `MediaHelper.createTestFile()` is asynchronous. 

What do you think? I know this way of defining blocks is a bit "legacy" and we now much prefer the block smoke testing model. I think I can make that work now, if you think it's better to do so - would be a fairly quick switch as I still have the draft files.

Reply:
> What do you think? I know this way of defining blocks is a bit "legacy" and we now much prefer the block smoke testing model.

That's totally fair, but I think it's fine to just add this to this existing spec. We can migrate these over in the future if we need, but this definitely works fine for now. 👍 

> but ran into difficulty because MediaHelper.createTestFile() is asynchronous.

Can you expand on this more? Where was the async nature causing problems?

Reply:
> Can you expand on this more? Where was the async nature causing problems?

It's because Jest doesn't allow the execution of async calls outside of the `describe/it` blocks, so I could not create a test video file _then_ pass the file path to the test file into the `configurationData` param.

Reply:
That said, I think I was able to come up with a workaround by passing in references to both the `TEST_VIDEO_FILE_PATH` constant _and_ the function `MediaHelper.createTestFile` separately into the `configurationData` object.